css/src/components/progress-bar.scss

```diff
@@ -0,0 +1,99 @@
+$progress-bar-height-medium: 2px;
+$progress-bar-height-large: 4px;
+
+$progress-bar-background: $secondary-background !default;
+$progress-bar-color: $primary !default;
+$progress-bar-success-color: $success !default;
+$progress-bar-warning-color: $warning !default;
+$progress-bar-danger-color: $danger !default;
+$progress-bar-border-radius-rounded: $border-radius-rounded !default;
+
+$progress-bar-animation-duration: 2s;
+
+.progress-bar-container {
+	position: relative;
+	width: 100%;
+	border-radius: $progress-bar-border-radius-rounded;
+	background-color: $progress-bar-background;
+	overflow: hidden;
+
+	.theme-high-contrast & {
+		background-color: $secondary-background-glow-high-contrast;
+	}
+
+	&.progress-bar-md {
+		height: $progress-bar-height-medium;
+	}
+
+	&.progress-bar-lg {
```

Comment:
Just wondering, why there are two sizes? I don't see Fluent proving size flexibility

Reply:
there are two sizes, medium and large in this doc: https://web-components.fluentui.dev/?path=/docs/components-progressbar--docs

Reply:
Lets play by the Figma file designers use - https://www.figma.com/design/By5idlaqQdmhTTnabMkuKk/Microsoft-Fluent-2-Web-(Community)?node-id=8934-59232&t=AHDqwimRDquM0BLB-0.